example-mojaloop-backend/values.yaml

```diff
@@ -426,65 +274,38 @@ ttksims-redis:
   ##
   architecture: standalone
 
+  ## @param fullnameOverride String to fully override  template
+  ##
+  fullnameOverride: ttksims-redis 
+
   ## Redis&reg; Authentication parameters
   ## ref: https://github.com/bitnami/containers/tree/main/bitnami/redis#setting-the-server-password-on-first-run
   ##
-  auth:
-    ## @param auth.enabled Enable password authentication
-    ##
-    enabled: false
+  # auth:
+  #   ## @param auth.enabled Enable password authentication
+  #   ##
+  #   enabled: false
 
   ## @section Persistence parameters
   ##
   master:
     ## Enable persistence using Persistent Volume Claims
     ## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
     ##
-    persistence:
-      ## @param persistence.enabled Enable MongoDB(&reg;) data persistence using PVC
-      ##
-      enabled: false
+    # persistence:
+    #   ## @param persistence.enabled Enable MongoDB(&reg;) data persistence using PVC
+    #   ##
+    #   enabled: false
 
     ## @param affinity Affinity for pod assignment
     ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
     ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
     ##
     affinity: {}
 
-    ## @param nodeSelector Node labels for pod assignment
-    ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
-    ##
-    # nodeSelector: {}
-    
-    ## Example
-    # nodeSelector:
-    #     "node-role.mojaloop.io": redis
-
-    ## @param tolerations Tolerations for pod assignment
-    ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
-    ##
-    # tolerations: []
-
-    ## Example
-    # tolerations:
-    #   - key: "node-role.mojaloop.io"
-    #     operator: "Equal"
-    #     value: redis
-    #     effect: "NoSchedule"
-
-  replica:
-    enabled: false
-    ## Enable persistence using Persistent Volume Claims
-    ## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
-    ##
-    persistence:
-      ## @param persistence.enabled Enable MongoDB(&reg;) data persistence using PVC
-      ##
-      enabled: false
-
 ## Reference: https://github.com/redpanda-data/console/blob/master/helm/console/values.yaml
 kafka-console:
-  enabled: true
+  enabled: false
```

Comment:
Why?

The dependency was also commented out, so this config technically no longer is useful.

Reply:
@mdebarros : per earlier comment I have taken kafka-console out for this PR